src/Orleans/GrainDirectory/MultiClusterRegistrationStrategy.cs

```diff
@@ -1,14 +1,33 @@
 ﻿using System;
+using System.Linq;
 using Orleans.Runtime.Configuration;
+using Orleans.MultiCluster;
+using System.Collections.Generic;
 
 namespace Orleans.GrainDirectory
 {
+
+    /// <summary>
+    /// Interface for multi-cluster registration strategies. Used by protocols that coordinate multiple instances.
+    /// </summary>
+    public interface IMultiClusterRegistrationStrategy {
+
+        /// <summary>
+        /// Determines which remote clusters have instances.
+        /// </summary>
+        /// <param name="mcConfig">The multi-cluster configuration</param>
+        /// <param name="myClusterId">The cluster id of this cluster</param>
+        /// <returns></returns>
+        IEnumerable<string> GetRemoteInstances(MultiClusterConfiguration mcConfig, string myClusterId);
```

Comment:
Intellisense glitch? At first I did not see any either, but now it reports 6 references. For example, in OrleansEventSourcing\Common\PrimaryBasedLogViewAdaptor.